app/models/goal.rb

```diff
@@ -0,0 +1,9 @@
+class Goal < ApplicationRecord
+  belongs_to :user
+
+  enum status: { todo: 0, progress: 1, done: 2 }
```

Comment:
maybe we can add another status `cancelled`, just a suggestion!

Reply:
Thank you for the suggestion, but I prefer not to display goals on the user's index page if they're not intentional or desired by the user.